.travis.yml

```diff
@@ -18,5 +18,12 @@ addons:
 cache:
   directories:
     - node_modules
+notifications:
+  email: false
 before_script:
   - npm prune
+after_success:
+  - npm run semantic-release
+branches:
+  except:
+    - /^v\d+\.\d+\.\d+$/
```

Comment:
Yeah, the branches part is a bit misleading. The idea here is to avoid running the tests when a new tag is pushed (one is pushed for every release). With semantic release the version number tags always correspond to a commit that was tested with a merge anyway, so it just cuts down on duplicate work that Travis has to do, speeding things up.